desk/app/activity.hoon

```diff
@@ -426,17 +461,24 @@
   |=  =(pole knot)
   ^+  cor
   ?+  pole  ~|(bad-watch-path+pole !!)
```

Comment:
```suggestion
  |=  =(pole knot)
  ^+  cor
  =?  pole  !?=([?(%v0 %v1) *] pole)
    [%v0 pole]
  ?+  pole  ~|(bad-watch-path+pole !!)
```

That way you don't have to handle the "no version prefix" case explicitly everywhere, looks/feels cleaner.

Reply:
Same for `+peek`.